invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/RelatedWorksField/RelatedWorksField.js

```diff
@@ -13,113 +13,178 @@ import {
+          disabled={disabled}
           showEmptyValue={showEmptyValue}
         >
           {({ arrayHelpers, indexPath }) => {
             const fieldPathPrefix = `${fieldPath}.${indexPath}`;
 
             return (
-              <GroupField optimized>
-                <SelectField
-                  clearable
-                  fieldPath={`${fieldPathPrefix}.relation_type`}
-                  label={i18next.t("Relation")}
-                  aria-label={i18next.t("Relation")}
-                  optimized
-                  options={options.relations}
-                  placeholder={{
-                    role: "option",
-                    content: "Select relation...",
-                  }}
-                  required
-                  width={3}
-                />
+              <Overridable
+                id="InvenioRdmRecords.DepositForm.RelatedWorksField.Item"
+                optimized
+              >
+                <GroupField optimized>
+                  <Overridable
+                    id="InvenioRdmRecords.DepositForm.RelatedWorksField.Relation"
+                    clearable
+                    optimized
+                    options={options.relations}
+                    required
+                    width={3}
+                  >
+                    <SelectField
+                      clearable
+                      fieldPath={`${fieldPathPrefix}.relation_type`}
+                      label={i18next.t("Relation")}
+                      aria-label={i18next.t("Relation")}
+                      optimized
+                      options={options.relations}
+                      placeholder={{
+                        role: "option",
+                        content: "Select relation...",
+                      }}
+                      required
+                      width={3}
+                    />
+                  </Overridable>
 
-                <TextField
-                  fieldPath={`${fieldPathPrefix}.identifier`}
-                  label={i18next.t("Identifier")}
-                  required
-                  width={4}
-                />
+                  <Overridable
+                    id="InvenioRdmRecords.DepositForm.RelatedWorksField.Identifier"
+                    required
+                    width={4}
+                  >
+                    <TextField
+                      fieldPath={`${fieldPathPrefix}.identifier`}
+                      label={i18next.t("Identifier")}
+                      required
+                      width={4}
+                    />
+                  </Overridable>
 
-                <SelectField
-                  clearable
-                  fieldPath={`${fieldPathPrefix}.scheme`}
-                  label={i18next.t("Scheme")}
-                  aria-label={i18next.t("Scheme")}
-                  optimized
-                  options={options.scheme}
-                  required
-                  width={2}
-                />
+                  <Overridable
```

Comment:
Wouldn't it just make sense to pass an overridable id to all these types of fields and define a dynamic overridable tag inside the SelectField and it would reduce the amount of code.